wishlist page/wishlist.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HomePage</title>
+    <link rel="stylesheet" href="/homepage/homepage.css">
+    <link rel="stylesheet" href="/product listing page/products.css">
+    <link rel="stylesheet" href="wishlist.css"> <link rel="stylesheet" href="../css/base.css">
+    <link rel="stylesheet" href="../css/spacing.css">
+    <link rel="stylesheet" href="https://galaxia-designs.netlify.app/styles.css" />
+    <script src="https://kit.fontawesome.com/7f4b9645a7.js" crossorigin="anonymous"></script>
```

Comment:
Consider importing this directly into your main CSS file with the CDN link, thus you wont have to keep importing it for each HTML file